src/main/java/bio/terra/service/snapshotbuilder/ConceptChildrenQueryBuilder.java

```diff
@@ -0,0 +1,44 @@
+package bio.terra.service.snapshotbuilder;
+
+import bio.terra.service.snapshotbuilder.query.FieldPointer;
+import bio.terra.service.snapshotbuilder.query.FieldVariable;
+import bio.terra.service.snapshotbuilder.query.Literal;
+import bio.terra.service.snapshotbuilder.query.Query;
+import bio.terra.service.snapshotbuilder.query.TableNameGenerator;
+import bio.terra.service.snapshotbuilder.query.TablePointer;
+import bio.terra.service.snapshotbuilder.query.TableVariable;
+import bio.terra.service.snapshotbuilder.query.filtervariable.BinaryFilterVariable;
+import bio.terra.service.snapshotbuilder.query.filtervariable.SubQueryFilterVariable;
+import java.util.List;
+
+public class ConceptChildrenQueryBuilder {
+
+  public static String buildConceptChildrenQuery(
```

Comment:
I made a separate file for this but it could just live in SnapshotBuilderService. thoughts?